src/pages/password/verified.tsx

```diff
@@ -0,0 +1,128 @@
+import { useEffect, useState } from 'react';
+import { useRouter } from 'next/router';
+import { css, Theme } from '@emotion/react';
+
+import LoadingHandler from '~/components/common/LoadingHandler';
+import { FixedSpinner } from '~/components/common/Spinner';
+import useCheckPasswordResetEmailVerifiedMutation from '~/hooks/api/auth/useCheckPasswordResetEmailVerifiedMutation';
+import useSendResetPasswordMutation from '~/hooks/api/member/useSendResetPasswordMutation';
+import { useToast } from '~/store/Toast';
+
+export default function PasswordResetVerified() {
+  const { fireToast } = useToast();
+  const { query } = useRouter();
```

Comment:
```suggestion
  const queryEmail = useRouterQuery('email', String);
  
  // 하단의 query.email => queryEmail로
```

아래 as를 이용해 타입을 보장하는 부분을 대체할 수 있을 거 같아요 !!

Reply:
useRouterQuery('email', String) 두 번째인자에 Number이런 형식으로 넣으면 자동으로 parser가 도나보내요!! 저런거 112처리하는거 귀찬았은데 좋은 거 같아요!

Reply:
오 좋습니다!! 이런건 리퀘체인지 날려주셔도 돼요! ㅋㅋㅋㅋㅋ